crates/ruff_python_parser/src/string.rs

```diff
@@ -110,99 +130,101 @@ impl<'a> StringParser<'a> {
             _ => std::char::from_u32(p).ok_or(unicode_error),
         }
     }
+    fn parse_octet(&mut self, o: u8) -> char {
+        let mut radix_bytes = [o, 0, 0];
+        let mut len = 1;
```

Comment:
nit: for me this feels more like an index variable rather than a length. I would prefer `idx` / `index` but if you think `len` is more suitable, it's totally fine.

Reply:
Yeah, I initially named it `idx`; however, the variable could be `3` if all of the octets are set in the escape. `3` would not be a valid index, so I named it `len`.